zerver/tests/test_signup.py

```diff
@@ -1752,6 +1752,17 @@ def test_create_realm_with_custom_language(self) -> None:
         self.assertEqual(realm.string_id, string_id)
         self.assertEqual(realm.default_language, realm_language)
 
+        # Check welcome messages
+        for stream_name, text, message_count in [
+            (Realm.DEFAULT_NOTIFICATION_STREAM_NAME, "con l'argomento", 3),
```

Comment:
Would be better to this text in a variable for readability like `italian_translation_for_private_stream` or something similar.